pkg/kn/commands/trigger/update_flag.go

```diff
@@ -0,0 +1,48 @@
+// Copyright © 2019 The Knative Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package trigger
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+)
+
+// TriggerUpdateFlags are flags for create and update a Trigger
+type TriggerUpdateFlags struct {
+	Broker string
+	Filter []string
+}
+
+// GetFilter to return a map type of filters
+func (f *TriggerUpdateFlags) GetFilter() map[string]string {
+	filters := map[string]string{}
+	for _, item := range f.Filter {
```

Comment:
And the plural is specifically for this kind of code where it reads much better when `f.Filters` is plural. Also, call the `item` by the singular, that is, `filter`

Reply:
fixed